packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -66,30 +67,36 @@ const classes = computed(() => {
     created: {
       icon: DocumentIconPlus,
       iconClasses: 'icon-dark-jade-400 icon-light-jade-50',
-      showTooltip: false,
     },
     modified: {
       icon: DocumentIconPlusMinus,
       iconClasses: 'icon-dark-orange-400 icon-light-orange-50',
-      showTooltip: false,
     },
     unmodified: {
       icon: CommitIcon,
       iconClasses: 'icon-light-gray-500',
-      showTooltip: true,
     },
   }[props.gql?.statusType || 'unmodified']
 })
 
-const gitTooltipSubtext = computed(() => {
+const tooltipMainText = computed(() => {
+  switch (props.gql?.statusType) {
+    case 'unmodified': return props.gql?.subject
```

Comment:
what does this props.gql?.subject show?

Reply:
This is the commit subject. I think #21139 can further show previous work updated in this PR.